FSJ_django20_project/FSJ/forms_adjudicator.py

```diff
@@ -0,0 +1,24 @@
+from .models_adjudicator import Adjudicator
+from django.forms import ModelForm
+
+class AdjudicatorForm(ModelForm):
+
+    class Meta:
+        model = Adjudicator
+        exclude = ()
+        
+    def __init__(self, *args, **kwargs):
+        super(AdjudicatorForm, self).__init__(*args, **kwargs)               
+        
+    
+class AdjudicatorRestrictedForm(ModelForm):
```

Comment:
Why not subclass AdjudicatorForm to simply disable fields already being set on the AdjudicatorForm? Unless we need to have different fields even exist between the two forms it is probably far easier to have the restricted form inherit from, and disable on initialization?

Reply:
This is true if Adjudicators can see all fields and simply cannot edit them, however, we had a long discussion about the merits of showing/hiding certain fields as opposed to simply disabling them. If we are hiding them, not sure I see the point of inheritance when we're simply going to override every part anyways.

Reply:
Yeah, if we expect that edit person can see fields viewing person cannot see, inheritance won't be as useful. If we're showing the same fields in both cases and simply disabling fields, we should consider inheritance. For some reason, I thought we would want to show the same subset of fields in both cases and only control enabling/disabling them.